overlord/snapstate/handlers.go

```diff
@@ -444,7 +444,7 @@ func (m *SnapManager) doDownloadSnap(t *state.Task, tomb *tomb.Tomb) error {
 	meter := NewTaskProgressAdapterUnlocked(t)
 	targetFn := snapsup.MountFile()
 
-	dlOpts := &store.DownloadOptions{}
+	dlOpts := &store.DownloadOptions{IsAutoRefresh: snapsup.IsAutoRefresh}
```

Comment:
not really related to this but the if below is a bit strange and now is stranger, I would prefer things turned around:

    if !snapsup.IsAutoRefresh {
        rate = 0
     }
     dlOpts := &store.DownloadOptions{IsAutoRefresh: snapsup.IsAutoRefresh, RateLimit: rate}